pyedflib/edfreader.py

```diff
@@ -124,60 +157,69 @@ def _get_float(self, v):
                 result[i] = float(v[i])
         return result
 
-    def _convert_string(self,s):
+    # FIX: Seems s must always be bytes because string has no decode method
+    def _convert_string(self, s: Union[bytes, str]) -> str:
         if isinstance(s, bytes):
             return s.decode("latin")
         else:
-            return s.decode("utf_8", "strict")
+            return s.decode("utf_8", "strict")  # type: ignore
```

Comment:
could you change to `elif isinstance(s, str): return s`?

Reply:
Done